src/Datadog.Trace/Logging/LibLogScopeEventSubscriber.cs

```diff
@@ -261,7 +261,7 @@ private void SetLogContext(ulong traceId, ulong spanId)
             }
         }
 
-        private void SetSerilogCompatibleLogContext(ulong traceId, ulong spanId)
+        private void SetSerilogCompatibleLogContext(TraceId traceId, ulong spanId)
```

Comment:
This line of code is only a DD feature for now, so we should make sure the string extracted here is in a decimal representation

Reply:
This also applies to the `SetLogContext` method

Reply:
64bit version should now preserve the old behaviour (decimal format).